configurations/default/env.schema.json

```diff
@@ -0,0 +1,172 @@
+      "description": "The OTP server's endpoint for plan."
+    },
+    "OTP_TIMEZONE": {
+      "type": "string",
+      "default": "America/Los_Angeles",
+      "description": "The timezone identifier that OTP is using to parse dates and times. OTP will use the timezone identifier that it finds in the first available agency to parse dates and times."
+    },
+    "MAXIMUM_PERMITTED_MONITORED_TRIPS": {
+      "type": "integer",
+      "default": "5",
+      "description": "The maximum number of saved monitored trips."
+    },
+    "MONGO_DB_NAME": {
+      "type": "string",
+      "default": "otp_middleware",
+      "description": "The name of the OTP Middleware Mongo DB."
+    },
+    "MONGO_HOST": {
+      "type": "string",
+      "default": "localhost:27017",
+      "description": "Mongo host address."
+    },
+    "MONGO_PROTOCOL": {
+      "type": "string",
+      "default": "mongodb",
+      "description": "Mongo DB protocol"
+    },
+    "NOTIFICATION_FROM_EMAIL": {
+      "type": "string",
+      "default": "noreply@email.com",
+      "description": "The from email address used in notification emails"
+    },
+    "NOTIFICATION_FROM_PHONE": {
+      "type": "string",
+      "default": "+15551234",
+      "description": "The from phone number used in notification SMSs"
+    },
+    "NOTIFICATION_TIME_FORMAT": {
+      "type": "string",
+      "default": "HH:mm",
+      "description": "The time format used in notification emails and SMSs."
+    },
+    "SENDGRID_API_KEY": {
+      "type": "string",
+      "default": "your-api-key",
+      "description": "Sendgrid settings available at: https://app.sendgrid.com/"
+    },
+    "SERVICE_DAY_START_HOUR": {
+      "type": "integer",
+      "default": "3",
+      "description": "Optional parameter for the hour (local time, 24-hr format) at which a service day starts. To make the service day change at 2am, enter 2. The default is 3am."
+    },
+    "SPARKPOST_KEY": {
+      "type": "string",
+      "default": "your-api-key",
+      "description": "Get Sparkpost key at: https://app.sparkpost.com/account/api-keys"
+    },
+    "TWILIO_ACCOUNT_SID": {
+      "type": "string",
+      "default": "your-account-sid",
+      "description": "Twilio settings available at: https://twilio.com/user/account"
+    },
+    "TWILIO_AUTH_TOKEN": {
+      "type": "string",
+      "default": "your-auth-token",
+      "description": "Twilio settings available at: https://twilio.com/user/account"
+    },
+    "VALIDATE_ENVIRONMENT_CONFIG": {
+      "type": "boolean",
+      "default": "true",
+      "description": "If set to true the environment configuration is validated against this schema."
+    }
+  },
+  "required": [
+    "AUTH0_DOMAIN",
+    "AUTH0_API_CLIENT",
+    "AUTH0_API_SECRET",
+    "BUGSNAG_API_KEY",
+    "BUGSNAG_ORGANIZATION",
+    "BUGSNAG_PROJECT_NOTIFIER_API_KEY",
+    "DEFAULT_USAGE_PLAN_ID",
+    "OTP_API_ROOT",
+    "OTP_TIMEZONE",
+    "MONGO_DB_NAME",
+    "VALIDATE_ENVIRONMENT_CONFIG"
```

Comment:
Is this line needed? It is redundant with `getConfigPropertyAsText("VALIDATE_ENVIRONMENT_CONFIG", "true")` in `ConfigUtils#validateConfig`.

Also, This is producing this message which seems to contradict itself:
```log
WARN  2020-12-21T09:51:43.878 [main] o.o.middleware.utils.ConfigUtils(ConfigUtils.java:169)  Config property VALIDATE_ENVIRONMENT_CONFIG not found. Using defaultValue: true.
ERROR 2020-12-21T09:51:44.133 [main] o.o.middleware.utils.ConfigUtils(ConfigUtils.java:90)  Unable to validate environment configuration.
java.lang.IllegalArgumentException: com.github.fge.jsonschema.core.report.ListProcessingReport: failure
--- BEGIN MESSAGES ---
error: object has missing required properties (["VALIDATE_ENVIRONMENT_CONFIG"])

```

Reply:
I have removed it from the schema's required list.